mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -2690,49 +2747,65 @@ struct Conv2DRewritePattern : public OpRewritePattern<T> {
           targetGemmDim1Attr.push_back(
               b.getNamedAttr("upper_layer_names",
                              b.getArrayAttr({b.getStringAttr(gemmKPad_name)})));
-        } else if (arg2TargetLayoutName2 == "gemmK") {
-          isOutputPad = true;
-          isGemmDim2Pad = true;
-          gemmDim2TargetName = b.getStringAttr(gemmKPad_name);
-
-          paddingOutputShape[2] = paddingOutputShape[2] + gemmKExtra;
-          sourceGemmDim2Attr.push_back(
-              b.getNamedAttr("transformation", b.getStringAttr("Pad")));
-          sourceGemmDim2Attr.push_back(b.getNamedAttr(
-              "parameters", b.getArrayAttr({b.getI32IntegerAttr(0),
-                                            b.getI32IntegerAttr(gemmKExtra)})));
-
-          targetGemmDim2Attr.push_back(
-              b.getNamedAttr("upper_layer_names",
-                             b.getArrayAttr({b.getStringAttr(gemmKPad_name)})));
+          // wrw
```

Comment:
please revise this comment per what you did for filter so it's more comprehensible.